core/app/models/uffizzi_core/credential.rb

```diff
@@ -2,4 +2,16 @@
 
 class UffizziCore::Credential < UffizziCore::ApplicationRecord
   include UffizziCore::Concerns::Models::Credential
+  extend Enumerize
+
+  enumerize :type,
```

Comment:
Please check it with platform.
You can copy `core` from uffizzi_app to uffizzi_platform and update Gemfile like this:
```
uffizzi_core path: './core'
```

Reply:
Apply some changes on platform [PR #293](https://github.com/UffizziCloud/uffizzi_platform/pull/293)